File: service/src/main/java/hu/bsstudio/team/model/Team.java
package hu.bsstudio.team.model;

import hu.bsstudio.entity.TeamType;
import java.util.List;
import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

@Value
@Builder
@Jacksonized
public class Team {
    long teamId;
    int year;
    String teamName;
    List<String> teamMembers;
    TeamType teamType;
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/team/model/Team.kt
package hu.bsstudio.robonaut.team.model

import hu.bsstudio.robonaut.entity.TeamType

data class Team(
  val teamId: Long = 0,
  val year: Int = 0,
  val teamName: String = "",
  val teamMembers: List<String> = listOf(),
  val teamType: TeamType = TeamType.JUNIOR,
)
